setup.cfg

```diff
@@ -25,6 +25,7 @@ omit =
     docs/modules/ROOT/examples/logging/tutorial003.py
     docs/modules/ROOT/examples/logging/tutorial004.py
     docs/modules/ROOT/examples/logging/tutorial005.py
+    docs/modules/ROOT/examples/testing/tutorial008.py
```

Comment:
```suggestion
```

Examples in docs showcase use cases of the system. We run tests against those examples to make sure they work as expected. `logging/tutorial003 - 005` examples were skipped from tests becase they use external systems like `rollbar` and `sentry`, which can not really be tested.

The point is we should run tests from `docs/modules/ROOT/examples/testing/tutorial008.py` in `tests.unit.test_docs.test_testing`. The test would look like following:
```python
def test_tutorial008(app_runner):
    tutorial008.test_reset_cache(app_runner)
    tutorial008.test_reuse_cache(app_runner)
```

Also after writing test above there is no need to have `test_reuse_cache` in `test_plugin`, which is a duplicate, so it can be removed.

Reply:
Done